ctru-rs/src/services/mod.rs

```diff
@@ -9,3 +9,48 @@ pub mod sslc;
 pub use self::apt::Apt;
 pub use self::hid::Hid;
 pub use self::sslc::SslC;
+
+use crate::Error;
+use std::sync::Mutex;
+pub(crate) struct ServiceHandler {
```

Comment:
nit: Using `Handler` in a type name usually means there is a better name that can be used instead. `ServiceCloser` or `ServiceReference` are some ideas.

Reply:
`ServiceCloser` is misleading, since it also starts the service.